components/organisation-unit-tree/types/index.d.ts

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react'
+
+export type OrganisationUnitTreeRoots = string | string[]
+
+interface OrganisationUnitNode {
+    id: string
+    displayName: string
+    children?: number
+    path: string
+}
+
+interface OrganisationUnitEventPayload extends OrganisationUnitNode {
+    checked: boolean
+    selected: string[]
+}
+
+export interface NodeLabelProps {
+    hasChildren: boolean
+    hasSelectedDescendants: boolean
+    loading: boolean
+    error: boolean
```

Comment:
This is app-runtime's [`FetchError`](https://github.com/dhis2/app-runtime/blob/master/services/data/src/engine/types/FetchError.ts), but I guess we have to use `Error`?

Reply:
Isn't this just a prop for controlling the error style?

Reply:
Depends on the provided `renderNodeLabel` prop, see [here](https://github.com/dhis2/ui/blob/0b1467396fea972eb4d471ee14b5bd8b7f11459f/components/organisation-unit-tree/src/organisation-unit-node/organisation-unit-node.js#L76).